mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/course/ChapterContentMenu.java

```diff
@@ -0,0 +1,44 @@
+package org.motechproject.nms.mobileacademy.domain.course;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+/**
+ * Contains the details about the menu file to be played at the end of the chapter
+ */
+@Entity
+public class ChapterContentMenu {
+
+    /**
+     * This is id for the End menu file of the chapter in the format”Chapter<ChapterId>_EndMenu”, where
+     * chapterId varies from 01 to 11.
+     */
+    @Field
+    private String identifier;
+
```

Comment:
This is used as a key into IMI's cms system, if I remember correctly and a mapping to their folder structure. This particular field is entirely related to their CMS and IVR menu and nothing to do with the course. 

It is kind of 'bookmarky' but not necessarily a bookmark for the course.